python/lsst/daf/persistence/repository.py

```diff
@@ -120,7 +120,7 @@ def _initMapper(self, repositoryCfg):
                     if mro[-1] is object:
                         mro = mro[:-1]
                     for c in mro:
-                        if arg in inspect.getfullargspec(c.__init__ ).args:
+                        if arg in inspect.getargspec(c.__init__ ).args:
```

Comment:
Yes, I will squash that. It's still a problem we need to fix but I'm not entirely sure how to fix it in the short term without a python version check. All that happens at the moment is you get a deprecation warning when you run any code.